conf/ocs_version/ocs-4.3.yaml

```diff
@@ -1,8 +1,5 @@
 ---
-RUN:
-  client_version: '4.3.0-0.nightly'
 DEPLOYMENT:
-  installer_version: "4.3.0-0.nightly"
   default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.3"
```

Comment:
Can we point to ocs-olm-operator, I remember ocs-registry is deprecated.

Reply:
done, I haven't noticed as I just removed just related things to this PR ;)